src/DotNetBridge/NativeDataView.cs

```diff
@@ -1470,10 +1454,10 @@ public override void CopyOut(long index, Batch batch, ref VBuffer<float> dst)
                     Contracts.Assert(0 <= index);
 
                     _getter(Data, ColIndex, index, null, null, true, out var size);
-                    var indices = dst.Indices;
+                    var indices = dst.GetIndices().ToArray();
```

Comment:
@yaeldekel, What is your recommendation here?  ToArray() will make a copy which is not what I would like unless I know dst is always going to be empty.

Reply:
I use VBufferEditor to avoid the extra .ToArray() memcopy

---
In reply to: [247203898](https://github.com/Microsoft/NimbusML/pull/77#discussion_r247203898) [](ancestors = 247203898)